lms/djangoapps/course_api/blocks/api.py

```diff
@@ -17,7 +17,9 @@ def get_blocks(
         requested_fields=None,
         block_counts=None,
         student_view_data=None,
-        return_type='dict'
+        return_type='dict',
+        block_types=None,
```

Comment:
Makes sense in terms of block_count. Thanks @nasthagiri. 
What did you have in mind for the potential refactor?